examples/math_func/config_advanced.yml

```diff
@@ -37,35 +35,30 @@ output_constraints:
 
 
 install_jobs:
-  -
-    name: adv_distance3
+  - name: adv_distance3
     source: jobs/ADV_DISTANCE3
-  -
-    name: adv_dump_controls
+  - name: adv_dump_controls
     source: jobs/ADV_DUMP_CONTROLS
 
 
 forward_model:
  # Compute distance (squared and negated) between 2 points
-  - adv_distance3     --point-file point.json
-                      --target-file data/r{{ realization}}/target.json
-                      --out distance
+  - adv_distance3     --point-file point.json --target-file data/r{{ realization}}/target.json
+    --out distance
```

Comment:
Is that valid yaml?, at least it looks ugly....

Reply:
I know it doesn't look pleasant, but it is valid yaml.